### ISSUE TITLE
Rename .github folder

### ISSUE THREAD

**Issue by tobias-wiese** [upb-uc4/api#10]
ATM you have a folder name ```.github/ISSUE_TEMPLATE``` in your repo. We (Theo and I) were not able to create a bug report with the new organization wide template. I think the naming is the reason for that.
Renaming it in ```.github``` with a sub folder called ```ISSUE_TEMPLATE``` containing your own templates should fix this issue

**Comment by bastihav** [comment]
We have a .github folder with a ISSUE_TEMPLATE folder inside..

**Comment by tobias-wiese** [comment]
Makes sense. It seems to be too late for me. Maybe the problem is that the folders in the ```.github``` repo and ```api/.github``` have the same name
